usaspending_api/settings.py

```diff
@@ -164,24 +165,27 @@
 }
 
 DATABASES = {
-    'default': {**dj_database_url.config(conn_max_age=10), **DEFAULT_DB_OPTIONS}
+    'default': {**dj_database_url.config(conn_max_age=CONNECTION_MAX_SECONDS), **DEFAULT_DB_OPTIONS}
 }
 
 # read replica env vars... if not set, default DATABASE_URL will get used
 # if only one set, this will error out (single DB should use DATABASE_URL)
 if os.environ.get('DB_SOURCE') or os.environ.get('DB_R1'):
-    DATABASES['db_source'] = dj_database_url.parse(os.environ.get('DB_SOURCE'), conn_max_age=10)
-    DATABASES['db_r1'] = dj_database_url.parse(os.environ.get('DB_R1'), conn_max_age=10)
+    DATABASES['db_source'] = dj_database_url.parse(os.environ.get('DB_SOURCE'), conn_max_age=CONNECTION_MAX_SECONDS)
+    DATABASES['db_r1'] = dj_database_url.parse(os.environ.get('DB_R1'), conn_max_age=CONNECTION_MAX_SECONDS)
     DATABASE_ROUTERS = ['usaspending_api.routers.replicas.ReadReplicaRouter']
 
 # import a second database connection for ETL, connecting to the data broker
 # using the environemnt variable, DATA_BROKER_DATABASE_URL - only if it is set
 if os.environ.get('DATA_BROKER_DATABASE_URL') and not sys.argv[1:2] == ['test']:
-    DATABASES['data_broker'] = dj_database_url.parse(os.environ.get('DATA_BROKER_DATABASE_URL'), conn_max_age=600)
```

Comment:
This will effectively change it from 600 seconds to 10 seconds. That seems like it will cause issues if someone deliberately put it to 600 for some reason.

Reply:
Very true. I cannot think of a good reason why the connection should remain for 600s after the last transaction completes